site/docs/types/type_variations.md

```diff
@@ -1,11 +1,11 @@
 # Type Variations
 
-Since Substrait is designed to work in both logical and physical contexts, there is a need to support extended attributes in the physical context. Different consumers may have multiple ways to present the same logical type. For example, an engine might support dictionary encoding a string or using either a row-wise or columnar representation of a struct. As such, there is the facility for specification users to express additional type variations for each logical type. These variations are expected to have the same logical properties as the canonical variation and are defined for each organization. The key properties of these variations are:
+Since Substrait is designed to work in both logical and physical contexts, there is a need to support extended attributes in the physical context. Different consumers may have multiple ways to present the same logical type. For example, an engine might support dictionary encoding a string or using either a row-wise or columnar representation of a struct. As such, there is the facility for users to express additional type variations for each logical type. These variations are expected to have the same logical properties as the canonical variation. They are defined as part of a YAML extension. The key properties of these variations are:
 
 | Property          | Description                                                  |
 | ----------------- | ------------------------------------------------------------ |
 | Base Type       | The base type this variation belongs to. Variations can only be expressed for simple types and wild-carded compound types (e.g. i8 or varchar(*)). |
-| Name              | The name used to reference this type. Should be unique within type variations for this parent type within an organization. |
+| Name              | The name used to reference this type. Should be unique within type variations for this parent type within a YAML extension. |
```

Comment:
I'd probably say "simple extension" here and elsewhere with a link to simple extension.

Reply:
Okay, I rewrote it a little bit (33636a1). I've gotten so used to calling them YAML extensions and protobuf `any` extensions in my head (because for validation anyway, simple extensions are anything but simple, and advanced extensions are anything but complicated) that I didn't even consider anymore that those are not the established names for them.